README.md

```diff
@@ -162,8 +163,9 @@ CLI commands are basically wrappers around the HTTP API of the server. The full
 hayhooks run     # Start the server
 hayhooks status  # Check the status of the server and show deployed pipelines
 
-hayhooks pipeline deploy-files <path_to_dir>   # Deploy a pipeline using PipelineWrapper
-hayhooks pipeline deploy <pipeline_name>       # Deploy a pipeline from a YAML file
+hayhooks pipeline deploy-files <path_to_dir>   # Deploy a pipeline using PipelineWrapper files (preferred)
+hayhooks pipeline deploy-yaml <path_to_yaml>   # Deploy a pipeline from a YAML file
+hayhooks pipeline deploy <path_to_dir>         # Alias for deploy-files
```

Comment:
I believe this is not true anymore since lower down I see we output a warning

Reply:
Good catch, thanks!